Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1388,11 +1396,20 @@ def poll_offense_events_with_retry(client: Client, search_id: str, offense_id: i
     failure_message = ''
     while num_of_failures <= max_retries:
         try:
+            print_debug_msg(f"Getting search status for {search_id}")
             search_status_response = client.search_status_get(search_id)
+            print_debug_msg(f"Got search status for {search_id}")
             query_status = search_status_response.get('status')
             # failures are relevant only when consecutive
             num_of_failures = 0
-            if query_status in TERMINATING_SEARCH_STATUSES:
+            print_debug_msg(f'Search query_status: {query_status}')
```

Comment:
I am a little afraid those many debug messages in the futures flow will really hurt performance
Imagine a customer defining to fetch more than THREADS COUNT 
because support_multithreaded is enabled, for every debug message only one will write each time
WDYT? @tkatzir @DeanArbel

Reply:
Why should support_multithreaded have an effect on log messages?
Isn't the logger multi threaded compatible?

The issue is that QRadar isn't "reliable" (for lack of a better term...) and we want to know about each call to it, especially during long_running.

Reply:
@Itay4 How does the logging mechanism happen? does it uses the Demisto Do or has a separate flow that the demisto.info/debug/error etc won't be affected by the lock?

Reply:
From what I can tell, it's:
QRadar_v3.print_debug_msg
demisto.debug

Since we `support_multithreading`, then we should be safe.

Reply:
all the log functions call `__do`, but not sure we should concerned with performance here - lets discuss